src/main/java/github/weichware10/toolbox/Main.java

```diff
@@ -1,71 +1,149 @@
 package github.weichware10.toolbox;
 
-import github.weichware10.toolbox.gui.ConfirmBox;
-import github.weichware10.toolbox.gui.Startbildschirm;
+import github.weichware10.toolbox.gui.TestVorbildschirm;
+import github.weichware10.util.config.ConfigClient;
+import github.weichware10.util.db.DataBaseClient;
+import io.github.cdimascio.dotenv.Dotenv;
 import java.io.IOException;
 import java.io.OutputStream;
-import java.io.PrintStream;
 import javafx.application.Application;
-import javafx.event.EventHandler;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
 import javafx.scene.Scene;
 import javafx.scene.control.TextArea;
 import javafx.scene.image.Image;
+import javafx.scene.text.Text;
+import javafx.stage.FileChooser;
 import javafx.stage.Stage;
-import javafx.stage.WindowEvent;
+import javafx.stage.FileChooser.ExtensionFilter;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'javafx.stage.FileChooser.ExtensionFilter' import. Should be before 'javafx.stage.Stage'.